src/main/java/hudson/tasks/junit/TestResultAction.java

```diff
@@ -69,7 +70,6 @@
 @SuppressFBWarnings(value = "UG_SYNC_SET_UNSYNC_GET", justification = "False positive")
 public class TestResultAction extends AbstractTestResultAction<TestResultAction>
         implements StaplerProxy, SimpleBuildStep.LastBuildAction {
-    private static final Logger LOGGER = Logger.getLogger(TestResultAction.class.getName());
```

Comment:
This was added in #625, but we already had a logger at the bottom of the file, it was just named lowercase `logger`.